website/common/locales/en/subscriber.json

```diff
@@ -14,6 +14,11 @@
     "mysteryItemText": "Each month you will receive a unique cosmetic item for your avatar! Plus, for every three months of consecutive subscription, the Mysterious Time Travelers will grant you access to historic (and futuristic!) cosmetic items.",
     "supportDevs": "Supports the developers",
     "supportDevsText": "Your subscription helps keep Habitica thriving and helps fund the development of new features. Thank you for your generosity!",
+    "giftSubscription": "Want to gift a subscription to someone?",
+    "giftSubscriptionText1": "Open their profile! You can do this by clicking on their avatar in your party header, or by clicking on their name in chat.",
```

Comment:
Grammar nit: maybe remove this comma since it does not separate 2 complete sentences?